internal/grpc/services/gateway/authprovider.go

```diff
@@ -75,6 +75,7 @@ func (s *svc) Authenticate(ctx context.Context, req *gateway.AuthenticateRequest
 			Status: res.Status,
 		}, nil
 	case res.Status.Code != rpc.Code_CODE_OK:
+		_ = status.NewErrorFromCode(res.Status.Code, "gateway")
```

Comment:
I think you can remove this line if you are not going to use the error.

Reply:
:+1: